client/src/components/PersonalInfo/index.js

```diff
@@ -0,0 +1,208 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import {
+  Grid,
+  Box,
+  Typography,
+  Input,
+  NativeSelect,
+  InputLabel,
+  FormControl,
+  TextField,
+  Button,
+} from '@material-ui/core';
+
+import LoaderProgress from '../../common-components/LoaderProgress';
+import Styles from './style';
+import Group from '../../assets/Group.svg';
+
+const PersonalInfo = () => {
+  const [information, setInformation] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [townName, settownName] = useState('');
+  const [cityName, setcityName] = useState('');
+  const [userStatus, setuserStatus] = useState('');
+  const [userNumber, setuserNumber] = useState('0');
+  const [deviceNumber, setdeviceNumber] = useState('0');
+  const [click, setClick] = useState(false);
+
+  const names = ['Halhul', 'Dura'];
+  const city = ['Hebron', 'Bethlehem'];
+  const status = ['None', 'single', 'Married'];
+  const handleChange = (event) => {
+    setClick(true);
+
+    settownName(event.target.value);
+  };
+  const handleCityChange = (event) => {
+    setClick(true);
+    setcityName(event.target.value);
+  };
+  const handleStatusChange = (event) => {
+    setClick(true);
+    setuserStatus(event.target.value);
+  };
+  const handleNumChange = (event) => {
+    setClick(true);
+    setuserNumber(event.target.value);
+  };
+  const handleDeviceChange = (event) => {
+    setClick(true);
+    setdeviceNumber(event.target.value);
+  };
```

Comment:
it is not the best practices, you can use one handle Change, and merge the status in one object